src/domain/community/inviteContributors/useInviteContributors.tsx

```diff
@@ -110,35 +109,25 @@ const useInviteContributors = ({
   };
 
   const [fetchAllVirtualContributors, { loading: accountVCsLoading }] = useAvailableVirtualContributorsLazyQuery();
-  const getAvailableVirtualContributors = async (filter: string | undefined, all: boolean = false) => {
+  const getAvailableVirtualContributors = async (filter: string | undefined) => {
     const { data } = await fetchAllVirtualContributors({
```

Comment:
I suggest wrapping this in `try/catch` since we're using `async` here.

Reply:
Hm, we have the global handler which would notify in case of an error. Do you have anything specific in mind?